src/gui/generalsettings.cpp

```diff
@@ -62,6 +82,7 @@ GeneralSettings::GeneralSettings(QWidget *parent)
     connect(_ui->newFolderLimitCheckBox, &QAbstractButton::toggled, this, &GeneralSettings::saveMiscSettings);
     connect(_ui->newFolderLimitSpinBox, static_cast<void (QSpinBox::*)(int)>(&QSpinBox::valueChanged), this, &GeneralSettings::saveMiscSettings);
     connect(_ui->newExternalStorage, &QAbstractButton::toggled, this, &GeneralSettings::saveMiscSettings);
+    connect(_ui->languageDropdown, QOverload<int>::of(&QComboBox::activated), this, &GeneralSettings::slotLanguagePicked);
```

Comment:
maybe be indexChanged?

As slotLanguagePicked is quite short, I'd prefer a lambda here :)